components/ActivityIndicator/examples/ActivityIndicatorTransitionExample.m

```diff
@@ -102,11 +102,11 @@ - (void)viewDidLoad {
   _refreshStrokeLayer.fillColor = [UIColor clearColor].CGColor;
   _refreshStrokeLayer.strokeColor = [UIColor blackColor].CGColor;
   _refreshStrokeLayer.strokeStart = 0;
-  _refreshStrokeLayer.strokeEnd = .8f;
+  _refreshStrokeLayer.strokeEnd = (CGFloat).8;
```

Comment:
tip: for consistency sake: `(CGFloat)0.8`

Reply:
Thanks. I'm trying to make minimal changes, so I didn't go through and prepend zeros to anything that didn't have it.  

@yarneo 
Do you think it's worth another pass to try and change all of them that match that pattern?